controllers/notes.controller.js

```diff
@@ -0,0 +1,94 @@
+const { Note } = require("../models/note.model");
+const { extend } = require("lodash");
+
+const getNotesByUserIdAndVideoId = async (req, res) => {
+  try {
+    const userId = req.get("userId");
+    const videoId = req.get("videoId");
```

Comment:
I am not able to understand req.get here. It might be req.params or req.body. correct me if I am wrong

Reply:
userid,videoid are coming in req headers . `req.get` is used to fetch the same !